index.bs

```diff
@@ -988,6 +992,11 @@ spec:css-syntax-3;
     1.  If |interfaces| is [=set/empty=], then return[=a promise rejected with=]
         a "{{NotSupportedError}}" {{DOMException}}.
 
+    1. If |options|.{{CredentialRequestOptions/mediation}} was not passed,
```

Comment:
Actually, I think if any of the interfaces default "required", then that becomes the default. 

Open question: could a spec default to anything other than "required" or "conditional"? (i.e., does a "silent" default ever make sense?)